src/main/java/umk/mat/pajda/ProjektZespolowy/DTO/TipDTO.java

```diff
@@ -0,0 +1,35 @@
+package umk.mat.pajda.ProjektZespolowy.DTO;
+
+import jakarta.validation.constraints.FutureOrPresent;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class TipDTO {
+
+    @NotNull
+    @Positive
+    private Integer id;
+
+    private String currency;
+
+    @NotNull
+    private Float amount;
+
+    @FutureOrPresent
+    private LocalDateTime paymentTime;
```

Comment:
createdAt is usually a better chocie for such name

Reply:
I got the variable name from database diagram. hmmmm, I agree with you, it;s better name, I will change that variable name